### ISSUE TITLE
Airsonic edge 11.0.0-SNAPSHOT.20210908073208 fails to start when upgrading from 10.6

### ISSUE THREAD

**Issue by khers** [airsonic-advanced/airsonic-advanced#617]
## Problem description

I am trying to update from Airsonic 10.6 to the latest edge release and the process is failing to start.

airsonic.log only contains
```
2021-09-10 15:33:44.222  INFO --- o.a.p.u.LegacyHsqlMigrationUtil          : HSQLDB database upgrade needed, from version 1.8.0 to 2.5
```

and the system journal says:
```
Sep 10 15:33:40 music systemd[1]: Started Airsonic Media Server.
-- Subject: A start job for unit airsonic.service has finished successfully
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
-- 
-- A start job for unit airsonic.service has finished successfully.
-- 
-- The job identifier is 6697.
Sep 10 15:33:43 music java[15680]:            _                       _
Sep 10 15:33:43 music java[15680]:      /\   (_)                     (_)
Sep 10 15:33:43 music java[15680]:     /  \   _ _ __  ___  ___  _ __  _  ___
Sep 10 15:33:43 music java[15680]:    / /\ \ | | '__|/ __|/ _ \| '_ \| |/ __|
Sep 10 15:33:43 music java[15680]:   / ____ \| | |   \__ \ (_) | | | | | (__
Sep 10 15:33:43 music java[15680]:  /_/    \_\_|_|   |___/\___/|_| |_|_|\___|
Sep 10 15:33:43 music java[15680]:                                                
Sep 10 15:33:43 music java[15680]:                         11.0.0-SNAPSHOT
Sep 10 15:33:44 music java[15680]: 2021-09-10 15:33:44.222  INFO --- o.a.p.u.LegacyHsqlMigrationUtil          : HSQLDB database upgrade needed, from version 1.8.0 to 2.5
Sep 10 15:33:44 music audit[15680]: SECCOMP auid=4294967295 uid=1003 gid=1003 ses=4294967295 pid=15680 comm="java" exe="/usr/lib/jvm/java-13-openjdk-amd64/bin/java" sig=31 arch=c000003e syscall=93 compat=0 ip=0x7fc82fdb94fb code=0x80000000
Sep 10 15:33:44 music kernel: audit: type=1326 audit(1631288024.259:43): auid=4294967295 uid=1003 gid=1003 ses=4294967295 pid=15680 comm="java" exe="/usr/lib/jvm/java-13-openjdk-amd64/bin/java" sig=31 arch=c000003e syscall=93 compat=0 ip=0x7fc82fdb94fb code=0x80000000
Sep 10 15:33:44 music systemd[1]: airsonic.service: Main process exited, code=killed, status=31/SYS
-- Subject: Unit process exited
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
-- 
-- An ExecStart= process belonging to unit airsonic.service has exited.
-- 
-- The process' exit code is 'killed' and its exit status is 31.
Sep 10 15:33:44 music systemd[1]: airsonic.service: Failed with result 'signal'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: http://www.ubuntu.com/support
-- 
-- The unit airsonic.service has entered the 'failed' state with result 'signal'.
```

### Steps to reproduce

1. *Take a working 10.6 install and stop the service*
2. *Drop the snapshoat war in place*
3. *Attempt to start the service*

## System information

 * **Airsonic version**: 11.0.0-SNAPSHOT.20210908073208 
 * **Operating system**: Ubuntu 20.04
 * **Java version**: openjdk version "13.0.7" 2021-04-20
OpenJDK Runtime Environment (build 13.0.7+5-Ubuntu-0ubuntu120.04)
OpenJDK 64-Bit Server VM (build 13.0.7+5-Ubuntu-0ubuntu120.04, mixed mode, sharing)
 * **Proxy server**: Apache 2.4.41 (Ubuntu)
 * **Client**: Can't get to the client at all
 * **Language**: en_US

**Comment by khers** [comment]
The issue came from using the suggested systemd unit file to start airsonic.  The list of system calls allowed does not include fchown (which is call 93 on amd64) and that was causing the signal.  I have worked around by removing the restriction on system calls.

Please close if appropriate.

**Comment by chris-schulz** [comment]
> The issue came from using the suggested systemd unit file to start airsonic. The list of system calls allowed does not include fchown (which is call 93 on amd64) and that was causing the signal. I have worked around by removing the restriction on system calls.
> 
> Please close if appropriate.

Hello,  would you mind sharing your startup script? Im running into high cpu usage issues.

**Comment by khers** [comment]
I started with this unit file: https://raw.githubusercontent.com/airsonic/airsonic/master/contrib/airsonic.service
but I have had to remove the sandboxing options because they haven't been updated and were causing airsonic to fail at startup.

I would be surprised if this is causing your high CPU usage problem

**Comment by randomnicode** [comment]
going to close this since it seems resolved

**Comment by Offebaescher** [comment]
I just want to add as a help for others: I came from subsonic and successfully managed to get version 10.6. running with my database format obviously still version 1.8.0
When I tried to move on to version 11 also on Ubuntu and also with the systemd unit files mentioned above, I ran into exactly the same error and could not solve it.

I then simply called it directly as root using

`
/usr/bin/java -Xmx700m -Dairsonic.home=/var/airsonic -Dserver.servlet.contextPath=/airsonic
 -Dserver.port=8080 -Dserver.forward-headers-strategy=native -jar /var/airson
ic/airsonic.war
`

It then successfully migrated the database and worked. I stopped it immediately again, made a "chown airsonic /var/airsonic -R" to correct the ownerships again as the upgrade and database conversion had generated a few folders and files which where then owned by root and since then airsonic again works fine being launched with the unit file running as user airsonic again.
